devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -38,6 +40,10 @@ metadata {
             input "checkInterval", "enum", title: "Presence timeout (minutes)", description: "Tap to set",
                     defaultValue:"2", options: ["2", "3", "5"], displayDuringSetup: false
         }
+        section {
```

Comment:
Yes, this device setting is absolutely crucial. Since this device setting can save power consumption, the parameter can't be 
configured with default value.

Reply:
Since "checkInterval" is already used in DTHs for healthCheck purposes. Please rename this setting. I think this one should be also handled in update() method.

Reply:
OK! I will rename this setting.